### DISCUSSION TITLE
Video de HTML y CSS

### DISCUSSION THREAD

**Discussion by YomairaBarrera** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#272] [Show and tell]
Hola Compañerxs para los que apenas vamos aprendiendo :smile: ,
Buscando documentación para poder hacer la practica de CSS encontré esta joya en YouTube, esta en ingles y dura 6 horas pero es una maravilla llevo 2.5 hrs y por fin entendí como funciona el box model del CSS.

https://www.youtube.com/watch?v=G3e-cpL7ofc&t=4522s&ab_channel=SuperSimpleDev

**Comment by OscarPedroza** [comment]
Muchas gracias por el aporte compañera!

**Comment by JuanSalvador73** [comment]
Gracias por el aporte.
